web/src/components/Meditor/Meditor.vue

```diff
@@ -301,6 +304,13 @@ const CommonVJSFOptions = computed(() => ({
   readOnlyPropertiesMode: 'hide',
 }));
 
+
+watchEffect(() => {
+  if (schema.value && editorInterface.value) {
+    validateDandisetMetadata(editorInterface.value)
```

Comment:
Since we're now doing validation manually with ajv, instead of lettings VJSF take care of it, do we need to disable VJSF's built in validation (so the two don't conflict)? I know of at least the `CommonVJSFOptions.initialValidation` param.

Reply:
That `initialValidation` param only controls whether or not VJSF runs validation on initialization, or if it waits for a change to the form. There doesn't seem to be a way to disable validation entirely. Which is not ideal, but the two shouldn't conflict because they're both using `ajv` against the same schema.